tests/modules/test_shrinkobjectstopoints.py

```diff
@@ -0,0 +1,101 @@
+import numpy
+import numpy.testing
+import pytest
+
+import cellprofiler.image
+import cellprofiler.measurement
+import cellprofiler.modules.shrinkobjectstopoints
+import cellprofiler.object
+
+
+instance = cellprofiler.modules.shrinkobjectstopoints.ShrinkObjectsToPoints()
+
+
+@pytest.fixture(scope="module")
+def image():
+    return cellprofiler.image.Image()
+
+
+@pytest.fixture(scope="module")
+def image_labels():
+    labels = numpy.zeros((20, 20), dtype=numpy.uint8)
+
+    labels[2:8, 2:8] = 1
+
+    labels[0:8, 12:18] = 2
+
+    labels[12:18, 0:8] = 3
+
+    labels[12:20, 12:20] = 4
+
+    return labels
+
+
+@pytest.fixture(scope="function")
+def object_set(objects):
+    object_set = cellprofiler.object.ObjectSet()
+
+    object_set.add_objects(objects, "InputObjects")
+
+    return object_set
+
+
+@pytest.fixture(scope="function")
+def objects():
+    return cellprofiler.object.Objects()
+
+
+@pytest.fixture(scope="module")
+def volume_labels():
+    labels = numpy.zeros((9, 20, 20), dtype=numpy.uint8)
+
+    labels[0:9, 2:8, 2:8] = 1
+
+    labels[0:5, 0:8, 12:18] = 2
+
+    labels[4:9, 12:18, 0:8] = 3
+
+    labels[1:8, 12:20, 12:20] = 4
+
+    return labels
+
+
+def test_resize_by_factor_shrink_image_labels(image_labels, module, object_set, objects, workspace):
```

Comment:
I don't understand `resize_by_factor` here. Should the test name be `test_shrink_image_labels`?